test/conftest.py

```diff
@@ -200,6 +202,9 @@ def pghoard(db, tmpdir, request):  # pylint: disable=redefined-outer-name
         "json_state_file_path": tmpdir.join("pghoard_state.json").strpath,
         "maintenance_mode_file": tmpdir.join("maintenance_mode_file").strpath,
     }
+    if db.ver == "10":
+        config["backup_sites"][test_site]["pg_xlog_directory"] = os.path.join(db.pgdata, "pg_wal")
```

Comment:
`pg_xlog_directory` default in config.py hasn't been adjusted for pg 10.  I'm not sure it makes a lot of sense to try to adjust it at all - we should just remove the config option per the comments in config.py and require `pg_data_directory` to be set instead (having no default value for it)